plugins/plugin-site/gatsby-config.mjs

```diff
@@ -28,6 +29,9 @@ Object.keys(process.env).forEach(key => {
     element[splits.slice(-1)[0]] = process.env[key];
 });
 
+const darkCodeTheme = fs.readFileSync('../../node_modules/github-syntax-dark/lib/github-dark.css', 'utf8');
```

Comment:
can this be done with require.resolve? or import.meta.resolve? https://stackoverflow.com/questions/54977743/do-require-resolve-for-es-modules

seeing node_modules in paths always make me nervous, and .../.../node_modules doubly so.